macros/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "macros"
```

Comment:
nit: could use another name - if there is a possibility for this to be published (ever?); can imagine there would be naming conflicts

Reply:
Sounds good. Done in 60918faa6ca6e1